unlock-app/package.json

```diff
@@ -10,7 +10,11 @@
     "@storybook/addon-viewport": "^4.1.13",
     "@storybook/addons": "^4.1.13",
     "@storybook/react": "4.1.13",
+    "@types/jest": "^24.0.11",
+    "@types/react": "^16.8.7",
+    "@types/react-redux": "^7.0.3",
```

Comment:
Types! These are additional packages that define types for these libraries. We'll probably do a lot of this, luckily just about every major JS library has a types package (whether official or community-run)